src/tag-editor/__tests__/get-tags-diff.test.tsx

```diff
@@ -0,0 +1,81 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { warnOnce } from '@cloudscape-design/component-toolkit/internal';
+
+import { TagEditorProps } from '../../../lib/components/tag-editor';
+import { getTagsDiff } from '../utils';
```

Comment:
Both imports should come from the same source, either the `src` or `lib` folder, but not mixed